### ISSUE TITLE
E/Pushwoosh: Can't get Notification: 

### ISSUE THREAD

**Issue by rommansabbir** [Pushwoosh/pushwoosh-android-sdk#145]
- Android Studio: 4.0.1
- minSdkVersion 17
- targetSdkVersion 28
- compileSdkVersion 28
- Emulator OS: Android 10

````
2020-08-20 12:48:15.888 18141-18360/com.elluminati.eber E/Pushwoosh: Can't get Notification: 
    java.lang.IllegalArgumentException: the bind value at index 2 is null
        at android.database.sqlite.SQLiteProgram.bindString(SQLiteProgram.java:167)
        at android.database.sqlite.SQLiteProgram.bindAllArgsAsStrings(SQLiteProgram.java:203)
        at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:49)
        at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1443)
        at android.database.sqlite.SQLiteDatabase.queryWithFactory(SQLiteDatabase.java:1290)
        at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1161)
        at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1329)
        at com.pushwoosh.repository.c.a(Unknown Source:29)
        at com.pushwoosh.repository.i.a(Unknown Source:10)
        at com.pushwoosh.notification.handlers.message.user.d.a(Unknown Source:4)
        at com.pushwoosh.notification.handlers.message.user.d.a(Unknown Source:36)
        at com.pushwoosh.notification.handlers.message.user.d.handleNotification(Unknown Source:38)
        at com.pushwoosh.notification.handlers.message.user.NotificationMessageHandler.handlePushMessage(Unknown Source:13)
        at com.pushwoosh.notification.d.a(Unknown Source:24)
        at com.pushwoosh.notification.NotificationServiceExtension.a(Unknown Source:75)
        at com.pushwoosh.notification.NotificationServiceExtension.handleMessage(Unknown Source:0)
        at com.pushwoosh.HandleMessageWorker.doWork(Unknown Source:51)
        at androidx.work.Worker$1.run(Worker.java:85)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
````

**Comment by wfhm** [comment]
@rommansabbir Could you please specify the Pushwoosh SDK version used in your project? Also, can you tell when exactly this crash occurs and describe the steps to reproduce the issue?

**Comment by rommansabbir** [comment]
When I'm trying to sending push from "Send Push" from " Applications/App/Send Push", I'm passing action to open activity with custom data. 

I noticed that when push sent from dashboard, on my class `FcmMessagingService` which  extends `NotificationServiceExtension` `onMessageReceived` called, after that the exception occur.

````
implementation 'com.pushwoosh:pushwoosh:6.0.4'
implementation 'com.pushwoosh:pushwoosh-firebase:6.0.4'
````

**Comment by wfhm** [comment]
@rommansabbir,

Do you generate a local notification in your FcmMessagingService class?

**Comment by rommansabbir** [comment]
Yes, but not notification code wasn't executed. I was just testing without creating notification.

**Comment by wfhm** [comment]
@rommansabbir,

Got it reproduced, will be fixed with the next release. It is just a warning though, it does not crash your app or block creating a notification in the Notification Center.

**Comment by wfhm** [comment]
Fixed in 6.0.5.
